sites/all/modules/reol_app_feeds/src/Helper/ParagraphHelper.php

```diff
@@ -745,10 +752,21 @@ private function getVideoBundleVideoList(\ParagraphsItemEntity $paragraph) {
     // eReolen uses field_video; eReolen Go uses field_breol_video.
     $videoUrl = $this->nodeHelper->getFieldValue($video, 'field_video', 'uri')
       ?? $this->nodeHelper->getFieldValue($video, 'field_breol_video', 'uri');
+    // eReolen uses field_link_color; eReolen Go uses field_material_carousel_color.
+    $color = $this->nodeHelper->getFieldValue($paragraph, 'field_link_color', 'rgb')
+      ?? $this->nodeHelper->getFieldValue($paragraph, 'field_material_carousel_color', 'rgb');
     $url = $this->nodeHelper->getFileUrl($videoUrl);
 
     $thumbnail = $this->getVideoThumbnail($url);
 
+    $carousels = array_map(static function (array $carousel) {
+      return array_filter($carousel, static function ($key) {
+        return in_array($key, ['query', 'title']);
+      }, ARRAY_FILTER_USE_KEY) + [
+        'type' => 'carousel',
+      ];
+    }, $this->getCarousel($paragraph));
```

Comment:
Hehehe array magic :-D

Reply:
No, it's just programming. I'll add a comment on what's actually going on.